dev/test/extension/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "azureextensiondev",
```

Comment:
What's with this file?

Reply:
It's a fake extension since these are npm packages, but the tests sometimes need to act as if they're running in the context of a VS Code extension. The UI and appservice packages already have one https://github.com/microsoft/vscode-azuretools/tree/master/ui/test/extension